### PR TITLE
Stripping fuxt of _get

### PR DESCRIPTION
**Pull request by Sveb**
Wasn't sure how to write getStripped util and GetOne util.

### CODE REVIEW

File: components/WpGutenberg.vue [hunk 1]
```diff
@@ -86,14 +81,13 @@ export default {
                 // Shape any props as needed
                 switch (output.componentName) {
                     case "gutenberg-image":
-                        output.image = _get(output, "mediaItem.nodes[0]", {})
+                        output.image = output.mediaItem?.nodes[0] || {}
```
Comment (drewbaker) [inline components/WpGutenberg.vue:84]: This should be:

```
output.mediaItem?.nodes?.[0]
```

Please fix anywhere you have array syntax please

File: components/WpImage.vue [hunk 2]
```diff
@@ -193,48 +193,49 @@ export default {
...
-            return _get(this, "image.mediaDetails.width", "auto")
+            return this.image.mediaDetails?.width || "auto"
         },
         parsedSrc() {
-            return this.src || _get(this, "image.sourceUrl", "")
+            return this.src || this.image.sourceUrl || ""
         },
         parsedSrcset() {
-            return this.srcset || _get(this, "image.srcSet", "")
+            return this.srcset || this.image.srcSet || ""
```
Comment (drewbaker) [inline components/WpImage.vue:209]: This should be:

```
this.image?.srcSet || ""
```

File: components/WpImage.vue [hunk 3]
```diff
@@ -193,48 +193,49 @@ export default {
...
         },
         parsedSizes() {
-            return this.sizes || _get(this, "image.sizes", "")
+            return this.sizes || this.image.sizes || ""
         },
         parsedColor() {
             return (
-                this.backgroundColor ||
-                _get(this, "image.imageMeta.primaryColor", "")
+                this.backgroundColor || this.image.imageMeta?.primaryColor || ""
```
Comment (drewbaker) [inline components/WpImage.vue:216]:
```
                this.backgroundColor || this.image?.imageMeta?.primaryColor || ""
```

File: components/gutenberg/Gallery.vue [hunk 4]
```diff
@@ -50,16 +52,12 @@ export default {
...
             ]
         },
         croppedRatio() {
             let output = null
-            const firstImageDimensions = _get(
-                this,
-                "images[0].mediaDetails",
-                {}
-            )
+            const firstImageDimensions = this.images[0]?.mediaDetails || {}
```
Comment (drewbaker) [inline components/gutenberg/Gallery.vue:60]:
```
            const firstImageDimensions = this.images?.[0]?.mediaDetails || {}
```

File: components/WpImage.vue (hunk 2 above)
Comment (Sveb) [reply components/WpImage.vue:209]: I'm guessing you want me to fix all of these computed items, not tjust the srcSet one.

...like so?

` parsedSizes() {
            return this.sizes || this.image?.sizes || ""
        },
         parsedVideoUrl() {
            return this.videoUrl || this.image?.imageMeta?.videoUrl || ""
        },
I'm going to leave it as it was plus the changes you requested, until tonight to double check with you.

Comment (drewbaker) [reply components/WpImage.vue]: Yes please fix all

File: components/WpImage.vue [hunk 5]
```diff
@@ -193,48 +193,51 @@ export default {
...
         parsedVideoUrl() {
-            return this.videoUrl || _get(this, "image.imageMeta.videoUrl", "")
+            return this.videoUrl || this.image?.imageMeta?.videoUrl || ""
         },
         parsedFocalPoint() {
             return {
                 x:
-                    _get(this, "focalPoint.x", false) ||
-                    _get(this.image, "imageMeta.focalPointX", ""),
+                    (this.focalPoint.x, false) ||
```
Comment (drewbaker) [inline components/WpImage.vue:227]:
```
this.focalPoint?.x
```

File: components/WpImage.vue [hunk 6]
```diff
@@ -193,48 +193,51 @@ export default {
...
+                    this.image?.imageMeta?.focalPointX ||
+                    "",
                 y:
-                    _get(this, "focalPoint.y", false) ||
-                    _get(this.image, "imageMeta.focalPointY", "")
+                    (this.focalPoint.y, false) ||
```
Comment (drewbaker) [inline components/WpImage.vue:231]:
```
this.focalPoint?.y
```

**Review by drewbaker — CHANGES_REQUESTED** [review]
Some more ones you missed

**Pull request merged**